addons/wowl/static/src/components/dropdown/dropdown_item.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+  <li t-name="wowl.DropdownItem" t-on-click.prevent="fire" owl="1">
```

Comment:
you have nada to prevent

Reply:
And if it replaces a stop we put earlier, let's make sure it's not broken on the multi-level (can't remember if you said it was safe or unknown)

Reply:
After our IRL discussion, I have stopped propagation instead of preventing default.
In this case, stopping propagation is better because we replace the clickevent with a custom business event.